### PR TITLE
taped-script flooder

### PR DESCRIPTION
**Pull request by fixxxedpoint**
- easy execution of aws flooder tests
- using prometheus/node_exporter for collecting performances metrics

### CODE REVIEW

File: fabfile.py [hunk 1]
```diff
@@ -240,6 +252,93 @@ def upgrade_binary(conn):
     # 3. restart binary
     conn.run(f'dtach -n `mktemp -u /tmp/dtach.XXXX` sh /home/ubuntu/cmd.sh')
 
+# ======================================================================================
+#                                       flooder
```
Comment (kostekIV) [inline fabfile.py:256]: nit: there is flooder section already.

**Review by kostekIV — APPROVED** [review]
LGTM

**Review by mike1729 — APPROVED** [review]

File: shell.py [hunk 2]
```diff
@@ -436,6 +437,31 @@ def upgrade_binary(regions, tag='dev', delay=0):
...
+
+
+def setup_flooder(n_flooders, regions, instance_type, tag):
+    flood_tag = tag+"-flood"
+
+    nhpr = n_parties_per_regions(n_flooders, regions)
+    launch_new_instances(nhpr, instance_type, 8, flood_tag)
+
+    color_print('waiting for transition from pending to running')
+    wait('running', regions, flood_tag)
```
[the remark this answers is not in the document]
Comment (fixxxedpoint) [reply shell.py]: removed

File: fabfile.py (hunk 1 above)
Comment (fixxxedpoint) [reply fabfile.py]: changed name

**Pull request merged**